src/content/dom/tips/imageTips.ts

```diff
@@ -31,7 +31,16 @@ export const imageTips = (el: Element): ElementTip[] => {
           result.push({ type: "error", content: "messages.noAltImage" });
         }
       } else {
-        result.push({ type: "error", content: "messages.noName" });
+        // FIXME: more roles should be checked
+        const ancestorControls = el.closest(
+          "a[href], button, [role=button], [role=link]",
```

Comment:
It seems good to refer [Presentational Children](https://www.w3.org/TR/wai-aria-1.2/#childrenArePresentational).